.swiftlint.yml

```diff
@@ -0,0 +1,10 @@
+disabled_rules:
+  - trailing_whitespace
```

Comment:
Look at this: https://github.com/realm/SwiftLint/issues/18

It seems xcode can be configured to handle this: https://github.com/realm/SwiftLint/issues/18